src/main/java/org/kohsuke/github/GHRepository.java

```diff
@@ -855,10 +870,25 @@ public void addCollaborators(GHUser... users) throws IOException {
      * @throws IOException
      *             the io exception
      */
-    public void addCollaborators(Collection<GHUser> users) throws IOException {
+    @Deprecated
+    public void addCollaborators(List<GHUser> users) throws IOException {
         modifyCollaborators(users, "PUT");
     }
 
+    /**
+     * Add collaborators.
+     *
+     * @param users
+     *            the users
+     * @param perm
+     *            the permission level
+     * @throws IOException
+     *             the io exception
+     */
+    public void addCollaborators(GHOrganization.Permission perm, Collection<GHUser> users) throws IOException {
```

Comment:
```
    public void addCollaborators(Collection<GHUser> users, GHOrganization.Permission perm) throws IOException {
```
Since `perm` is optional, from an API design view I want it to be on the end, but then the `(GHOrganization.Permission perm, GHUser... users)` signature looks out of place.  

What's your opinion? Do we just take this as it is or reverse this and remove the other signature?

Reply:
That's a tough question.  I agree with the API design view of having permissions at the end, but if we get rid of the other signature then anyone that only needs to add a single collaborator will need to create and populate a collection.  Since there is already a method that takes a collection, what do you think about changing the signature to only accept a single user?

`public void addCollaborator(GHUser user, GHOrganization.Permission permission)`

Reply:
Or we just accept that one signature is out of place but convenient and appropriate.

Reply:
Sounds good.